Create pt-study-group.md (#2)
OpenScienceLabs/request-forms: .github/ISSUE_TEMPLATE/pt-study-group.md

[![Forms](https://www.muz.ifsuldeminas.edu.br/images/2019/02/oficina_google/1.png)](https://docs.google.com/forms/d/e/1FAIpQLSdfd1ZzbPKmZ0aWaAnfnvY9aW2iXkz1pTzyvuOm1gcKczwafQ/viewform?usp=sf_link)

# Solicitação para criação de um Grupo de Estudos - _Open Science Labs_

> ## **Regras**

Reviewer: `##` ja é para cabeçalho. Nao necessitas por em negritas `**`

- Todas as perguntas do formulário devem ser respondidas corretamente e devem corresponder ao Grupo de Estudos que deseja criar, qualquer discrepância entre as respostas e os encontros são passíveis de encerramente do mesmo. Caso queira modificar alguma informação, como alterar o dia dos encontros, entrar em contato prévio com os gestores.

- Os tutores estabelecidos no formulário têm total responsabilidade pelo Grupo de Estudos e devem gerir os encontros com respeito e inclusão. O ingresso das pessoas ao Grupo é aberto à todos que gostariam de participar e nenhuma limitação pode ser imposta aos participantes.

- As datas definidas devem ser cumpridas e o cancelamento/adiamento de um dos horários deve ser comunicado previamente aos participantes do Grupo de Estudos.

- Os materiais apresentados não podem ser provenientes de pirataria, plataformas com assinatura ou conter conteúdo inapropriado, ofensivo ou desrespeitoso.

> # **Formulário**

Reviewer: Same here

---

> name: Forms to create a Study Group

Reviewer: change it to

```
name: New Study Group Request
```

> about: Fill in this form to create a Study Group on Discord Server

Reviewer: ```suggestion
about: Request Form for the creation of a new Study Group on OSL Discord
```

> title: 01/22/22 - Solicitação para criação de um Grupo de Estudos - Open Science Labs

Reviewer: this title will be generated automatically ... so we shouldn't have something fixed
maybe you change it to:

```
title: "[DATE]: [FEATURE NAME] <STUDY GROUP NAME>" 
```
